src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -162,13 +166,15 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
 
         JsonUserPrefsStorage userPrefsStorage =
                 new JsonUserPrefsStorage(temporaryFolder.resolve("ExceptionUserPrefs.json"));
-        StorageManager storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        JsonEventManagerStorage eventManagerStorage =
```

Comment:
Good implementation, separating the JSON files could reduce complexity in parsing